Backend/Notification/Email.cs

```diff
@@ -11,6 +14,78 @@ public class Email
         public string Body { get; set; }
         public string Subject { get; set; }
 
+        public Email()
+        {
 
+        }
+
+        public void AddAttachmentToTemplate(MailMessage message)
+        {
+            using (message)
+            {
+                Attachment mac = new Attachment(@"../../../EmailTemplate/images/home -background.png");
+                message.Attachments.Add(mac);
+                string contentID = "Mac";
+                mac.ContentId = contentID;
+
+                Attachment swapify = new Attachment("../../../EmailTemplate/images/swapify.png");
+                message.Attachments.Add(swapify);
+                string contentID2 = "Swapify";
+                swapify.ContentId = contentID2;
+                
+                swapify.ContentDisposition.Inline = true;
+                swapify.ContentDisposition.DispositionType = DispositionTypeNames.Inline;
+                mac.ContentDisposition.Inline = true;
+                mac.ContentDisposition.DispositionType = DispositionTypeNames.Inline;
+            }
+        }
+
+        public string GetRegistrationEmailSubject()
+        {
+            return Subject = "Registrácia na SWAPIFY";
+        }
+
+        public string GetRegistrationEmailBody(string baseUrl,string confirmLink)
+        {
+            using (var reader = new StreamReader("../../../EmailTemplate/RegistrationTemplate.html"))
```

Comment:
this will not work. You should copy the templates to output directory and then load it from there.
StreamReader is throwing some exceptions. You should handle them and log error if some exception occurred.